### DISCUSSION TITLE
Error in filebeat logs un sure of

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#8614] [Unsupported Versions]
I am seeing this in my filebeat logs every 10 seconds.
`Error running input: error receiving slowlog data: dial tcp 127.0.0.1:6379: connect: connection refused`

**Comment by dougburks** [comment]
Seems like Filebeat is trying to connect to Redis which seems unusual for our default configurations. Have you modified any settings?

**Comment by intsec** [comment]
I have been fine tuning and configuring this for a little while but nothing that I can think of that dealt with Redis and Filebeat.  Any place I should be looking?

**Comment by dougburks** [comment]
If the Filebeat logs are saying that Filebeat is trying to connect to Redis, then I would double-check the Filebeat configuration and perhaps compare it to a fresh installation.
